PreemptRtCheck.pm

```diff
@@ -33,19 +33,29 @@ sub executeWithKernelConfig($)
 	my $self = shift;
 	my $kernelConfig = shift;
 
-	if ( $kernelConfig !~ /CONFIG_PREEMPT_RT=y|CONFIG_PREEMPT_RT_FULL=y/)
+	if ( $kernelConfig =~ /CONFIG_PREEMPT_RT=y|CONFIG_PREEMPT_RT_FULL=y/)
 	{
-		$self->{RESULTKIND} = "not good";
-		$self->{RESULT} = "not found";
-		$self->{COMMENT} = "Kernel without real-time capabilities found\n".
-			"For more information, see http://wiki.linuxaudio.org/wiki/system_configuration#installing_a_real-time_kernel";
-	}
-	else
-	{	
 		$self->{RESULTKIND} = "good";
 		$self->{RESULT} = "found";
 		$self->{COMMENT} = undef;
+		return
 	}
+
+	my $cmdline = `cat /proc/cmdline`;
```

Comment:
the the kernel init cmdline the only way to enable threadirq's?

Reply:
I believe so, yes. I also checked the Kernel Config already being used in this Check and didn't see anything indicating the parameter in there. Based on what I've found, looking at the init cmdline is the only way to verify this parameter's existence.

FWIW this is also basically (although semantically implemented slightly differently) the solution mxa suggested in #31

Reply:
Great, thanks!